src/EnergyPlus/CurveManager.hh

```diff
@@ -571,6 +571,20 @@ namespace CurveManager {
 	int
 	GetCurveObjectTypeNum( int const CurveIndex ); // index of curve in curve array
 
+	void
+	checkCurveValue(
+		std::string const callingRoutineObj, // calling routine with object type
+		std::string const objectName,     // parent object where curve is used
+		int const curveIndex,             // index to curve object
+		std::string const cFieldName,     // object field name
+		std::string const cFieldValue,    // user input curve name
+		Real64 const Var1,                // required 1st independent variable
+		Optional <Real64 const > Var2,    // 2nd independent variable
```

Comment:
@rraustad I think you can avoid the need for trailing "_" in the calls by providing a default here
`Optional <Real64 const > Var2 = "_", // 2nd independent variable`
Like [CurveManager.hh](https://github.com/NREL/EnergyPlus/blob/173bb58bcc95ec005c91946ecee8f573224904fd/src/EnergyPlus/CurveManager.hh#L412-L418)